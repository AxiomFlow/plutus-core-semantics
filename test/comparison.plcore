(module Foo (imported) (exported () ()) (local () ())
  (define c1 (builtin lessThanInt 1 2))
  (define c2 (builtin lessThanInt 2 2))
  (define c3 (builtin lessThanFloat 0.1 1.0))
  (define c4 (builtin lessThanFloat 1.0 0.1))
)
