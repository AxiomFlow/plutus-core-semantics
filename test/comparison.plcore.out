<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . c3 ) |-> ( con Nat . True .ValList ) ( Foo . c4 ) |-> ( con Nat . False .ValList ) ( Foo . c1 ) |-> ( con Nat . True .ValList ) ( Foo . c2 ) |-> ( con Nat . False .ValList ) </genv> <mod> Foo </mod> <sym> .Map </sym> </T>
